Cargo.toml

```diff
@@ -21,8 +21,8 @@ rpath = true
 rpath = true
 
 [workspace.dependencies]
-alloy-eips = { version = "0.11.0", default-features = false }
-alloy-serde = { version = "0.11.0", default-features = false}
+alloy-eips = { version = "0.8.0", default-features = false }
+alloy-serde = { version = "0.8.0", default-features = false}
```

Comment:
I'm wondering why the downgrade here? This can be a problem when merging `main` into `feat/solidity-tests` as the Foundry crates are only tested to work with REVM 19.4 and `alloy-* = 0.11`

Reply:
When enabling the `blst` feature flag, which is required to support this EIP, I was getting `cargo` errors due to incompatible versions of `blst` due to dynamically linking the dependency. I had to revert to the same `alloy` versions REVM is using to guarantee that the code could compile.

Reply:
Ah I see thanks for the explanation! Could you try to upgrade to `alloy-* = 0.11.1`, `revm = 19.5` and `revm-precompile=16.1.0`? Those should work together based on the Foundry `Cargo.lock`:

https://github.com/foundry-rs/foundry/blob/f3130a52937039a7e82b4aef07143139c322d09f/Cargo.lock

Reply:
I will try; thanks!

Reply:
That worked: https://github.com/NomicFoundation/edr/pull/808/commits/7bb29d226458f38624923bfbc83b76755a70bccf

Thanks for finding a fix that doesn't cause any issues on your branch! :D